.env.example

```
@@ -22,3 +24,5 @@ NEXT_PUBLIC_SENTRY_DSN=
 ARCJET_KEY=
 
 RESEND_API_KEY=
+
+ALLOWED_TRANSCRIPT_HOSTS="ohio.stream-io-cdn.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)